data/json/recipes/food/brewing.json

```diff
@@ -244,10 +244,17 @@
     "difficulty": 3,
     "time": "8 m",
     "batch_time_factors": [ 50, 4 ],
-    "autolearn": true,
+    "book_learn": [ [ "vinegar_maker", 1 ], [ "winemaking_beginner", 1 ] ],
     "components": [
       [ [ "water", 2 ], [ "water_clean", 2 ] ],
-      [ [ "juice", 2 ], [ "apple_cider", 4 ], [ "cheap_wine", 2, "LIST" ], [ "worthy_wine", 1, "LIST" ] ]
+      [
+        [ "juice", 2 ],
+        [ "apple_cider", 4 ],
+        [ "cheap_wine", 2, "LIST" ],
+        [ "worthy_wine", 1, "LIST" ],
+        [ "sweet_fruit", 1, "LIST" ],
+        [ "wild_herbs", 40 ]
+      ]
```

Comment:
Is this included by accident? Probably shouldn't be changing the vinegar recipe in this PR.

Reply:
I changed it because the vinegar making book I found online specifically called out making vinegar with wild herbs. But let me know if it needs to revert.